ezflow/data/dataset/base_dataset.py

```diff
@@ -133,6 +137,15 @@ def __getitem__(self, index):
         img2 = torch.from_numpy(img2).permute(2, 0, 1).float()
         flow = torch.from_numpy(flow).permute(2, 0, 1).float()
 
+        if self.normalize:
+            #input_transform = transforms.Compose([
```

Comment:
Can we remove these comments if not required for future use?

Reply:
Comments have been removed.